spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```

Comment:
- [ ] Your migrations are giving some issues due to their order kindly re-order your migrations so next reviewer won't face any problem and shall test your controller properly 👍